zellij-server/src/panes/tiled_panes/mod.rs

```diff
@@ -871,7 +881,13 @@ impl TiledPanes {
                     *self.display_area.borrow(),
                     *self.viewport.borrow(),
                 );
-                let next_index = pane_grid.next_selectable_pane_id_to_the_left(&active_pane_id);
+                let mut next_index = pane_grid.next_selectable_pane_id_to_the_left(&active_pane_id);
+
+                if next_index.is_none() && wrap_around {
+                    let plugin_count = self.plugin_pane_count();
+                    let index = self.panes.len() - plugin_count - 1;
+                    next_index = Some(PaneId::Terminal(index as u32));
```

Comment:
I don't think this will always work. Maybe we can use `TiledPaneGrid` and `FloatingPaneGrid` to find the leftmost/rightmost pane in such cases?